llap-server/src/java/org/apache/hadoop/hive/llap/counters/WmFragmentCounters.java

```diff
@@ -119,6 +123,21 @@ public void dumpToTezCounters(TezCounters tezCounters, boolean isLast) {
     for (int i = 0; i < fixedCounters.length(); ++i) {
       tezCounters.findCounter(LlapWmCounters.values()[i]).setValue(fixedCounters.get(i));
     }
+
+    // add queue and runtime (together with task count) on a "per daemon" level
+    // to the Tez counters.
+    if (addTaskTimeCounters) {
+      String hostName = MetricsUtils.getHostName();
+      long queued = fixedCounters.get(LlapWmCounters.GUARANTEED_QUEUED_NS.ordinal())
+                    + fixedCounters.get(LlapWmCounters.SPECULATIVE_QUEUED_NS.ordinal());
+      long running = fixedCounters.get(LlapWmCounters.GUARANTEED_RUNNING_NS.ordinal())
+                     + fixedCounters.get(LlapWmCounters.SPECULATIVE_RUNNING_NS.ordinal());
+
+      CounterGroup cg =tezCounters.getGroup("LlapTaskRuntimeAgg by daemon");
```

Comment:
i think style is off here i see a missing space after `=`

Reply:
Done